docs/content/NavList.mdx

```diff
@@ -325,9 +183,11 @@ If a `NavList.Item` contains a `NavList.SubNav`, the `NavList.Item` will render
 
 ### With a divider
 
-```jsx
+```jsx live drafts
 <NavList>
-  <NavList.Item href="/">Dashboard</NavList.Item>
+  <NavList.Item href="/" aria-current="page">
+    Dashboard
+  </NavList.Item>
   <NavList.Item href="/pulls">Pull requests</NavList.Item>
   <NavList.Item href="/issues">Issues</NavList.Item>
   <NavList.Divider />
```

Comment:
This is interesting 🤔 

Would we also want to support `<NavList showDivider={true}` [similar to Action list](https://primer.style/react/ActionList#:~:text=can%20be%20selected.-,showDivider,-boolean)?

Reply:
I don't think dividers in between individual items is part of the NavList design cc @vdepizzol